contracts/core/OptimisticLightClient.sol

```diff
@@ -48,83 +51,94 @@ contract OptimisticLightClient is ILightClient {
     /**
      * @inheritdoc ILightClient
      */
-    function addOpConsensusState(
-        L1Header calldata l1header,
-        OpL2StateProof calldata proof,
-        uint256 height,
-        uint256 appHash
-    ) external override returns (uint256 fraudProofEndTime, bool ended) {
-        uint256 hash = consensusStates[height];
+    function updateClient(bytes calldata proof, uint256 peptideHeight, uint256 peptideAppHash) external override {
```

Comment:
`peptide` is only our execution engine. I would suggest we use `polymerHeight` here which is more generic and does not reveal internal details of our implementation.

Reply:
though if we have a different execution engine, won't we have a different light client contract? I was guessing that OptimisticLightClient is tied one-to-one with peptide hence the naming

Reply:
ahh ok, I guess this isn't necessarily true that we'd have to add a newlight client in this case, I changed!